### DISCUSSION TITLE
Ability to abort dangerous dialog button on hover lost

### DISCUSSION THREAD

**Discussion by Joehuu** [ppy/osu#22594] [Ideas] [answered]
https://github.com/Joehuu/osu/compare/abort-dangerous-dialog-button-on-hover-lost

I have this already implemented, but I guess the flow is to open a discussion for UX behavioral changes before PRing.

| Before | After |
| --- | --- |
| ![osu!_126Oe5xogk](https://user-images.githubusercontent.com/35318437/218010280-3ff4dad9-4822-46fe-8f5a-6b04bf76e530.gif) | ![osu!_FcF2L6deyU](https://user-images.githubusercontent.com/35318437/218010085-1c61b345-433f-47e6-9dbf-a71dbfd1cbd3.gif) |

**Comment by bdach** [comment, marked as answer]
Makes sense to me.

**Reply by Joehuu** [reply]
PRed in https://github.com/ppy/osu/pull/22604.
